.envrc

```diff
@@ -0,0 +1,15 @@
+if [[ `hostname` = "alicecerno2" ]] && [[ -z ${ROOT_BASEDIR} ]]; then
```

Comment:
```suggestion
if [[ $(hostname) == "alicecerno2" ]] && [[ -z ${ROOT_BASEDIR} ]]; then
```

Reply:
https://www.shellcheck.net/wiki/SC2006

Reply:
Ok, I'm old 😳

Reply:
Equality check inside `[[ ]]` should use `==`.